src/parser/jobs/rdm/GeneralCDDowntime.js

```diff
@@ -17,7 +17,7 @@ export default class GeneralCDDowntime extends CooldownDowntime {
 
 	constructor(...args) {
 		super(...args)
-		super.setAbilityList(TRACKEDCDS)
-		super.setDownTimeDuration(DOWNTIME_OK_TIME)
+		super.trackedcds = TRACKED_COOLDOWNS
+		super.downtimeOkTime = DOWNTIME_OK_TIME
```

Comment:
This doesn't ready very well - maybe `validDowntime` or `okDowntime` or something? I think it's the dupe 'time' that's throwing.